ingestion/src/metadata/ingestion/source/amundsen.py

```diff
@@ -223,7 +294,7 @@ def create_dashboard_entity(self, dashboard):
             service_entity = get_dashboard_service_or_create(
```

Comment:
instead of get or create, we should just GET?

Reply:
if in the docs we specify that users should first create the services, then IMO we should:
1. try to get the service
2. if it is not there, throw an exception calling out that "service named XYZ of type ABC cannot be found"

Reply:
I'm logging an error in case of service not found, shall we throw error 🤔

Reply:
mmh logging + ignoring the entity might be good enough!